.github/workflows/prettier.yml

```diff
@@ -9,6 +9,7 @@ on:
 
 jobs:
   prettier:
+    if: ${{false}}
```

Comment:
is this just to bypass it

Reply:
Yes, Sry should of let you know, im just debugging now